### ISSUE TITLE
Incorrect results

### ISSUE THREAD

**Issue by dutypro** [vormkracht10/laravel-seo-scanner#50]
The results seem to returning incorrect results.

`php artisan seo:scan-url https://www.bbc.co.uk/news`

Returns the following:

```
-----------------------------------------------------------------------------------------------------------------------------------
> https://www.bbc.co.uk/news | 15 passed 9 failed
-----------------------------------------------------------------------------------------------------------------------------------

✔ Robots.txt allows indexing
✔ The page does not have 'noindex' set
✔ The page does not have 'nofollow' set
✔ The page response returns a 200 status code
✔ All links redirect to an url using HTTPS
✔ The page contains no broken images
✔ Every image has an alt tag
✔ Length of the content is at least 2100 characters
✔ The page contains no broken links
✔ Too long sentence check
✔ CSS files are not bigger than 15 KB
✔ Time To First Byte (TTFB) is below 600 ms
✔ HTML is GZIP compressed
✔ Images are not larger than 1 MB
✔ Javascript files are not bigger than 1 MB

✘ The page has a meta description failed.
The page does not contain a description meta tag, while it should. Estimated time to fix: 1 minute(s).

✘ The lang attribute is set on the html tag failed.
The page does not contain a lang attribute, while it should. Estimated time to fix: 1 minute(s).

✘ The page has an Open Graph image failed.
The page does not contain an open graph image, while it should. Estimated time to fix: 20 minute(s).

✘ The page title is not longer than 60 characters failed.
The page does not contain a title tag, while it should. Estimated time to fix: 1 minute(s).

✘ The page has the focus keyword in the first paragraph failed.
The page does not contain the focus keyword in the first paragraph, while it should. Estimated time to fix: 10 minute(s).

✘ The page has an H1 tag and if it is used only once per page failed.
The page does not contain any h1 tag, while it should. Estimated time to fix: 1 minute(s).

✘ The page has the focus keyword in the title failed.
The page title does not contain the focus keyword, while it should. Estimated time to fix: 1 minute(s).

✘ Transition word ratio check failed.
The page contains too few transition words. The recommended minimum is 30%, while the actual number is 0%. Estimated time to fix: 60 minute(s).

✘ HTML is not larger than 100 KB failed.
The page contains HTML that is too large (max 100 KB), the actual size is 112.93 KB. Estimated time to fix: 60 minute(s).


Completed 24 out of 24 checks.
```
Looking at the page source at the URL, the site does have a meta description, HTML lang tag, OG image, etc.

**Comment by Baspa** [comment]
Thanks for mentioning this issue. I will take a look at it at and come back to you. Do you get incorrect results with your own website(s)? @dutypro

**Comment by Baspa** [comment]
I see that's failing the meta description tag because the website does not have a `<meta name="description" .. >` tag. So it's producing a correct result according to the check. However it does use have a `og:description` tag. We should probably check on those tags as well. 

If you want, you can make a PR to fix the checks. I could also make a PR but that can take up to 1 week..

**Comment by atmonshi** [comment]
hey
I am having the same issue, most of the checks giving worng result
I have to disable this in `Seo.php` class

```php
->withHeaders([
              //'Accept-Encoding' => 'gzip, deflate',
                ...$headers,
            ]);
```

to make it work
not sure why, I test it in prod and local.
would it be a good idea if this moved to the config?!


and thank you for the package :)

**Comment by Baspa** [comment]
So if I'm correct, when disabling the Encoding request header you're getting correct result @atmonshi? That seems odd.

**Comment by atmonshi** [comment]
@Baspa , yes very odd, 
np, I'll open a PR to move the Encoding to the config if that sound good?

**Comment by Baspa** [comment]
That would be great. But I think the headers are used in the compress check. So maybe disabling that check would be sufficient. Feel free to make a PR then we can check on that :) 

Have a good weekend.

**Comment by Baspa** [comment]
Seems like this had something to do with not decoding the response. That's why the checks failed, they got a encoded response. I changed this in #53, and am currently testing it. Planning on releasing this today.

**Comment by Baspa** [comment]
Fixed in `v3.6.0`
